src/main/java/core/basesyntax/service/serviceimpl/ReportServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.impl.StorageDaoImpl;
+import core.basesyntax.service.ReportService;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String TITLE = "fruit,quantity";
+    private static final String SEPARATOR = ",";
+    private final StorageDao storageDao = new StorageDaoImpl();
```

Comment:
The code should not be implementation dependent, fix everywhere
```suggestion
    private final StorageDao storageDao;
    
    public ReportServiceImpl(StorageDao storageDao) {
        this.storageDao = storageDao;
    }
```